### PR TITLE
enhancement in AllBroadcasts page

### PR DESCRIPTION
**Pull request by SarveshLimaye**
## Issue that this pull request solves

 Closes: #683 

## Proposed changes
To add month and year of the broadcast at the end of of each card
Search should take place correctly based on the title of the broadcast

### Brief description of what is fixed or changed

Added mont and year of broadcast at the end of each card.

Fixed search functionality

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
screencapture-localhost-3000-all-broadcasts-2022-01-02-22_50_50

screenrecording (1)

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/src/pages/Broadcast/Component/AllBroadcasts/AllBroadcasts.jsx [hunk 1]
```diff
@@ -167,7 +174,12 @@ export function AllBroadcasts(props) {
         </div>
       </div>
       <div id={style["all-cards"]}>
-        {array.map((element, i) => {
+        {!filteredItems.length && (
+          <div>
+            <h2>No broadcast found ..</h2>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/AllBroadcasts/AllBroadcasts.jsx:179]: Please change it to `<h1>No Broadcasts Found !</h1>`

File: frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx [hunk 2]
```diff
@@ -30,6 +30,25 @@ export function Card(props) {
...
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+  // console.log(date.getMonth());
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx:48]: Please remove these console statements

File: frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx [hunk 3]
```diff
@@ -48,6 +67,11 @@ export function Card(props) {
             <div className={style["card-content"]}>
               {props.project.content.substring(0, 400)}...
             </div>
+            <div className={style["card-content"]}>
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx:70]: Can we please make it to appear something like `- <month>, Year` for eg - `January, 2022` and display it at the right end of the card?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@SarveshLimaye please so the below requested changes-

The footer shifts up when there is lesser/no broadcasts. It should stick at the bottom of the page.
Screenshot 2022-01-03 at 1 56 20 PM

File: frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx (hunk 3 above)
Comment (SarveshLimaye) [reply frontend/src/pages/Broadcast/Component/AllBroadcasts/Card/Card.jsx:70]: Yeah, I will try to do it

**Comment by SarveshLimaye** [comment]
> @SarveshLimaye please so the below requested changes-
> 
> The footer shifts up when there is lesser/no broadcasts. It should stick at the bottom of the page. Screenshot 2022-01-03 at 1 56 20 PM

@Kajol-Kumari  done !

screenrecording (1)

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@SarveshLimaye after the filter by month&year changes got merged, your logic needs a little modification. Please look into the below screenshots for ref-

Screenshot 2022-01-04 at 6 17 51 PM
Screenshot 2022-01-04 at 6 16 54 PM

**Comment by SarveshLimaye** [comment]
@Kajol-Kumari done !

screenrecording (2)

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@SarveshLimaye Can we please fix the below two issues-
1. The footer shifts up when there is less/no filtered broadcasts. Can we please make it to appear at the bottom?
2. The message `No Broadcasts Found !` should appear in a single line. I can see a woed break currently. Can we please fix that as well?

Rest looks clean and good to go!

**Comment by SarveshLimaye** [comment]
Hey, @Kajol-Kumari I have fixed the word breaking issue in No Broadcast Found! . I am not able to solve the footer issue. Can you pls help me with it?

**Comment by Kajol-Kumari** [comment]
> Hey, @Kajol-Kumari I have fixed the word breaking issue in No Broadcast Found! . I am not able to solve the footer issue. Can you pls help me with it?

@SarveshLimaye you can provide the div containing message of `No Broadcasts Found` a margin of maybe 10% similar to-
```
#all-cards h1 {
  margin: 10%;
}
```

**Comment by SarveshLimaye** [comment]
@Kajol-Kumari Thanks a lot it worked . 

screenrecording-chunk-1

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for making the requested changes @SarveshLimaye 
LGTM 👍

**Pull request merged**